csm/core/services/s3/accounts.py

```diff
@@ -178,16 +178,8 @@ async def patch_account(self, account_name: str, password: str = None,
                 new_creds.secret_key_id, CsmS3ConfigurationFactory.get_iam_connection_config())
 
         if password:
-            # We will try to create login profile in case it doesn't exist
-            new_profile = await client.create_account_login_profile(account_name, password)
-            if isinstance(new_profile, IamError) and \
-                    new_profile.error_code != IamErrors.EntityAlreadyExists:
-                self._handle_error(new_profile)
-
-            if isinstance(new_profile, IamError):
-                # Profile already exists, we need to set new passord
-                Log.debug(f"Update Login Profile for account {account_name}")
-                new_profile = await client.update_account_login_profile(account_name,
```

Comment:
Please, apply the same logic to the IAM user PATCH request, i.e. remove creating IAM user login profile before setting the password.

Reply:
Please, do not resolve conversations yourself, it's reviewers job.